lib/active_merchant/billing/gateways/ixopay.rb

```diff
@@ -173,6 +176,17 @@ def add_preauth(xml, money, options)
         end
       end
 
+      def add_capture(xml, money, authorization, options)
+        currency = options[:currency] || currency(money)
+
+        xml.capture do
+          xml.transactionId          new_transaction_id
+          xml.referenceTransactionId authorization
```

Comment:
I think that this should be
```
xml.referenceTransactionId  authorization&.split('|')&.first
```
as done in the `refund` method above so that the customer is able to just pass in the same `authorization` that we returned to them.

Reply:
Makes sense. 👍